lib/matplotlib/projections/polar.py

```diff
@@ -373,12 +373,14 @@ def _wrap_locator_formatter(self):
         self.isDefault_majfmt = True
 
     def clear(self):
+        # docstring inherited
         super().clear()
         self.set_ticks_position('none')
         self._wrap_locator_formatter()
 
     @_api.deprecated("3.4", alternative="ThetaAxis.clear()")
     def cla(self):
+        """Clear the current axes."""
         self.clear()
```

Comment:
Since this was deprecated in 3.4, it can be removed in 3.6 which is our next release. So, you can remove this here if you want. You'd need to add a doc api_change_note for removals indicating that it has been removed too.